Vision/OpenCV/__app__/utilities/opencvskill.py

```diff
@@ -0,0 +1,46 @@
+import cv2
+from . import webapiskill
+import os
+
+class opencvskill:
+    def __init__(self, debug_override=False):
+        if debug_override or ("debugMode" in os.environ and os.environ["debugMode"] == "true"):
+            self.debug_state = []
+            self.debug_mode = True
+    
+    def __add_debug_step(self, debug_step):
+        if (self.debug_mode):
+            self.debug_state.append(debug_step)
+
+    def get_output_debug_steps(self):
+        result = []
+        for debug_step in self.debug_state:
+            is_success, result_image = cv2.imencode(".png", debug_step["image"])
+            result_debug_step = { "image": { "$type": "file", "name": "output.png", "contentType": "image/png", "data": webapiskill.encode_image_bytes(result_image) } }
```

Comment:
Do you want to make some of those parameters configurable through env variables or parameters?